shared/profile/index.native.js

```diff
@@ -104,7 +105,23 @@ class Profile extends Component<Props, State> {
     })
   }
 
+  _onClickAvatar = () => {
+    showImagePicker({mediaType: 'photo'}, (response: Response) => {
+      if (response.didCancel) {
+        return
+      }
+      if (response.error) {
+        console.error(response.error)
+        throw new Error(response.error)
+      }
+      this.props.onEditAvatar(response)
+    })
+  }
+
   _makeUserBio(loading: boolean) {
+    if (this.props.isYou && this.props.bioEditFns) {
+      this.props.bioEditFns.onEditAvatarClick = this._onClickAvatar
```

Comment:
we should never write to props

Reply:
I'm not sure what to do here. I need to use `react-native-image-picker`'s `showImagePicker`. So, I can't override it in the container (which seems like the right thing to do). The desktop fails to render with:

```
Failed to compile.

./node_modules/react-native-image-picker/index.js
Module parse failed: Unexpected token (22:2)
You may need an appropriate loader to handle this file type.
| 
| module.exports = {
|   ...ImagePickerManager,
|   showImagePicker: function showImagePicker(options, callback) {
|     if (typeof options === 'function') {
```